packages/atomic/create-atomic/src/fetch-page.ts

```diff
@@ -0,0 +1,44 @@
+import 'isomorphic-fetch';
+import 'abortcontroller-polyfill';
+import HttpsProxyAgent from 'https-proxy-agent';
+import {PlatformClient, IManifestResponse} from '@coveord/platform-client';
+
+export async function fetchPageManifest(
+  host: string,
+  organizationId: string,
+  pageId: string,
+  accessToken: string
+) {
+  const globalRequestSettings: Record<string, unknown> = {};
+  const proxyServer = process.env['https_proxy'] || process.env['HTTPS_PROXY'];
+  if (proxyServer) {
+    const httpsProxyAgent = HttpsProxyAgent(proxyServer);
+    globalRequestSettings.agent = httpsProxyAgent;
+  }
+
+  const client = new PlatformClient({
+    globalRequestSettings,
+    organizationId,
+    accessToken,
+    host,
```

Comment:
@louis-bompart Is using the host ok, or we need to map the platform url in some way? 

Also notice the use of PlatformClient + proxy management. It will take a bit till everything is ready but locally I manage to have the full loop working:
<img width="1115" alt="Screen Shot 2022-04-04 at 4 03 39 PM" src="https://user-images.githubusercontent.com/4923043/161624075-33067418-4126-4ee9-80a5-ec52f093c825.png">
<img width="726" alt="Screen Shot 2022-04-04 at 4 04 11 PM" src="https://user-images.githubusercontent.com/4923043/161624076-53eb7e96-2257-43a7-a630-525970989beb.png">
<img width="1128" alt="Screen Shot 2022-04-04 at 4 04 38 PM" src="https://user-images.githubusercontent.com/4923043/161624077-2c20e465-8feb-464a-bf01-a3a9ab3fb948.png">